### PR TITLE
✅ Fixes flaky test api-server #3569

### LINKED ISSUE #3569: Flaky tests/public-api/test_solvers_jobs_api.py::test_run_job[FAILED]
FAILED tests/public-api/test_solvers_jobs_api.py::test_run_job[FAILED] - osparc.exceptions.ApiException: (404)

[example](https://github.com/ITISFoundation/osparc-simcore/actions/runs/3477365541/jobs/5813586054#step:11:36)

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

-  ✅ ♻️ cleanup ``test/public-api``
- 🐛 fixes failing docker log dumps issue that avoided producing container log artifacts.
- Temporary **disables failing test** until logs policy in the director-v2 is clarified 

### Insight on Failing Test

- In the public-api tests, I run a sleeper that will exit with failure (there is a flag to force it). 
- Then I try to download the logsfile for the failing sleeper but it does not exits
- Tracing the issue i noticed that the directorv2 actually is deleting the logsfile (and the outputs) because the job failed https://github.com/ITISFoundation/osparc-simcore/blob/04af546b173d7a1c4201367f46d82657ab896d7a/services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/dask_scheduler.py#LL176C40-L176C40
-  @sanderegg: Why would you delete the logs of a failing run? Am I misinterpreting something? :-)

## Related issue/s

- fixes #3569

## How to test

```cmd
cd tests/public-api
make install-dev
pytest -vv test_solvers_jobs_api.py
```

### CODE REVIEW

File: packages/pytest-simcore/src/pytest_simcore/helpers/utils_docker.py [hunk 1]
```diff
@@ -14,6 +15,18 @@
 from tenacity.stop import stop_after_attempt
 from tenacity.wait import wait_fixed
 
+
+# SEE https://docs.docker.com/engine/api/v1.42/#tag/Container/operation/ContainerList
+class ContainerStatus(str, Enum):
```
Comment (sanderegg) [inline packages/pytest-simcore/src/pytest_simcore/helpers/utils_docker.py:20]: question: why don't you use the generated models in models-library?

**Review by sanderegg — APPROVED** [review]
ok cool. and let's check this question tomorrow.

File: packages/pytest-simcore/src/pytest_simcore/helpers/utils_docker.py (hunk 1 above)
Comment (pcrespov) [reply packages/pytest-simcore/src/pytest_simcore/helpers/utils_docker.py:20]: true

**Review by GitHK — APPROVED** [review]
👍

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**